tests/components/reolink/conftest.py

```diff
@@ -0,0 +1,62 @@
+"""Setup the Reolink tests."""
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+TEST_HOST = "1.2.3.4"
+TEST_HOST2 = "4.5.6.7"
+TEST_USERNAME = "admin"
+TEST_USERNAME2 = "username"
+TEST_PASSWORD = "password"
+TEST_PASSWORD2 = "new_password"
+TEST_MAC = "ab:cd:ef:gh:ij:kl"
+TEST_PORT = 1234
+TEST_NVR_NAME = "test_reolink_name"
+TEST_USE_HTTPS = True
+
+
+@pytest.fixture(name="reolink_setup_entry")
+def reolink_setup_entry_fixture() -> Generator[AsyncMock, None, None]:
```

Comment:
Could you use the standard name for this one, just for consistency with pretty much all other integrations?
```suggestion
@pytest.fixture
def mock_setup_entry() -> Generator[AsyncMock, None, None]:
```